test/integration_jenkins/plugin_dt_test.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+# Copyright 2021 Dynatrace LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'net/http'
+require 'fluent/test'
+require 'fluent/test/helpers'
+require 'fluent/test/driver/output'
+require 'fluent/plugin/out_dynatrace'
+require 'json'
+
+class TestPluginDynatraceIntegration < Test::Unit::TestCase
+  include Fluent::Test::Helpers
+
+  def active_gate_url
+    ENV["ACTIVE_GATE_URL"]
+  end
+
+  def api_token
+    ENV["API_TOKEN"]
+  end
+
+  def setup
+    Fluent::Test.setup
+  end
+
+  # default configuration for tests
+  def config
+    %(
+    active_gate_url #{active_gate_url}/api/v2/logs/ingest
+    api_token       #{api_token}
+    ssl_verify_none    true
```

Comment:
Add a comment why this might be required for private AGs (or make it configurable)